packages/protocol/contracts/governance/Governance.sol

```diff
@@ -615,30 +653,66 @@ contract Governance is
    * @notice Approves the hash of a hotfix transaction(s).
    * @param hash The abi encoded keccak256 hash of the hotfix transaction(s) to be approved.
    */
-  function approveHotfix(bytes32 hash) external hotfixNotExecuted(hash) onlyApprover {
-    hotfixes[hash].approved = true;
-    emit HotfixApproved(hash);
+  function approveHotfix(
+    bytes32 hash
+  ) external hotfixNotExecuted(hash) onlyApproverOrSecurityCouncil {
+    if (msg.sender == approver) {
+      hotfixes[hash].approved = true;
+    } else {
+      if (isL2()) {
+        hotfixes[hash].councilApproved = true;
```

Comment:
there should be check whether msg.sender == securityCouncil

Reply:
What would you suggest in terms of readability improvement?

Reply:
Probably removing modifier since we are checking approver twice and keep the logic only in this function.

Reply:
Not sure it improved readability, but I removed the modifier and included the logic in the function body.